tsconfig.json

```diff
@@ -4,18 +4,18 @@
         "esModuleInterop": true,
         "skipLibCheck": true,
         "target": "es2022",
-        "verbatimModuleSyntax": true,
+        "verbatimModuleSyntax": false,
         "allowJs": true,
         "resolveJsonModule": true,
         "moduleDetection": "force",
         /* Strictness */
         "strict": true,
         "noUncheckedIndexedAccess": true,
         /* If NOT transpiling with TypeScript: */
-        "moduleResolution": "Bundler",
+        "moduleResolution": "node",
         "module": "ESNext",
         "noEmit": true,
         /* If your code doesn't run in the DOM: */
-        "lib": ["es2022"]
+        "lib": ["dom", "es2022"]
```

Comment:
are these changes in tsconfig still required?

Reply:
Indeed `dom` and `node` are not necessary anymore. Good catch. resolved this in my last commit. However, `"verbatimModuleSyntax": false,` is still necessary as the test suit does not run otherwise. See error message below. Someting is wrong with the imports in case the option is set to `true`: 

Test suite failed to run
                                                                                                                                                                                                                                 
    src/modules/plural_voting.test.ts:1:10 - error TS1286: ESM syntax is not allowed in a CommonJS module when 'verbatimModuleSyntax' is enabled.                                                                                
                                                                                                                                                                                                                                 
    1 import { PluralVoting } from './plural_voting';

Reply:
thanks for the explanation